### PR TITLE
Update `expo-modules-autolinking` version

### PR DESCRIPTION
**Pull request by nandorojo**
# Why

The existing version is outdated and requires a resolution for dev clients to build. `expo run:ios` on SDK 45 doesn't work with the older auto-linking version.

# How

Evan, Tomasz, Cedric and Lukasz helped me find this solution in 🇵🇱

# Test Plan

I tested this in my app with a yarn resolution.

Related: https://github.com/expo/expo/issues/17923

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [x] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Comment by ide** [comment]
@tsapeta Could you please review this?

**Comment by tsapeta** [comment]
The `expo` package is usually released from release branches (e.g. `sdk-45`) and the latest version depends on the correct version of autolinking (see https://github.com/expo/expo/blob/aa4163b5cb533fd4b3fc583e3958b36eaa97617c/packages/expo/package.json#L67)

The issue that we encountered in your project was caused by `expo-splash-screen` depending on config plugins package that depends on older version of autolinking — this is something that @EvanBacon is going to fix as part of the versioned CLI.

Moreover, all CI jobs are failing with this change, because `expo-modules-autolinking` in the workspace is `0.7.0` as well (`0.8.x` were released from `sdk-45`).

I'm going to publish `expo`, `expo-modules-core` and `expo-modules-autolinking` pretty soon and this should fix package versions not being up to date on the main branch.

To sum up, this change won't fix anything so I'm closing the PR 😢

**Comment by nandorojo** [comment]
Got it, makes sense! Thanks for the explanation. I also had a feeling `main` wasn’t the right branch to point to, given that the version number was on beta.
